source/includes/fundamentals/code-snippets/records/DataStorageRecord.java

```diff
@@ -0,0 +1,8 @@
+package fundamentals.codecs.records;
+
+// start dataStorageRecord
+public record DataStorageRecord(
+        String productName,
+        Double capacity
```

Comment:
Unless you intend for capacity to be optional, this could be a `double`

Reply:
Do you plan to change this?

Reply:
Yes, sorry, didn't check in the file.